cmd/build-push-image/main.go

```diff
@@ -139,15 +143,28 @@ func main() {
 	if err == nil {
 		fmt.Println("Image exists already.")
 	} else {
-
 		fmt.Printf("Building image %s ...\n", imageName)
-		err := buildahBuild(opts, imageRef, os.Stdout, os.Stderr)
+		err = buildahBuild(opts, imageRef, os.Stdout, os.Stderr)
 		if err != nil {
 			log.Fatal("buildah bud: ", err)
 		}
 
+		extraTagsSpecified, err := shlex.Split(opts.extraTags)
+		if err != nil {
+			log.Fatalf("parse extra tags (%s): %s", opts.extraTags, err)
+		}
+		extraTags := []string{}
+		if len(extraTagsSpecified) > 0 {
+			log.Printf("extra tags: %+q", extraTagsSpecified)
+		}
+		for _, tagSpecified := range extraTagsSpecified {
+			extraTags = append(extraTags, fmt.Sprintf(
+				"%s/%s/%s:%s",
+				opts.registry, imageNamespace, imageStream, tagSpecified))
+		}
```

Comment:
Maybe we can extract that into a function that maps the extraTags option to the `extraTags` slice?

Reply:
This has changed quite a bit. So this is likely no longer relevant.